client/modules/IDE/utils/consoleStyles.js

```diff
@@ -0,0 +1,189 @@
+  OBJECT_VALUE_NULL_COLOR: '#FFA9D9',
+  OBJECT_VALUE_UNDEFINED_COLOR: '#FFA9D9',
+  OBJECT_VALUE_REGEXP_COLOR: '#2DE9B6',
+  OBJECT_VALUE_STRING_COLOR: '#2DE9B6',
+  OBJECT_VALUE_SYMBOL_COLOR: '#B3BEFF',
+  OBJECT_VALUE_NUMBER_COLOR: '#FFA9D9',
+  OBJECT_VALUE_BOOLEAN_COLOR: '#FFA9D9',
+  OBJECT_VALUE_FUNCTION_KEYWORD_COLOR: '#F5DC23',
+  LOG_ERROR_BACKGROUND: '#1F0000',
+  LOG_ERROR_COLOR: '#EA7B7D',
+  LOG_WARN_BACKGROUND: 'hsl(50, 100%, 10%)',
+  LOG_WARN_COLOR: '#F5BC38',
+  LOG_INFO_COLOR: '#D9D9D9',
+  LOG_INFO_BORDER: '#4D4D4D',
+  LOG_COLOR: '#D9D9D9',
+  LOG_DEBUG_COLOR: '#38B6F5',
+  LOG_DEBUG_BACKGROUND: '#05232E',
+  LOG_DEBUG_BORDER: '#0C546E',
+  TABLE_BORDER_COLOR: 'grey',
+  TABLE_TH_BACKGROUND_COLOR: 'transparent',
+  TABLE_TH_HOVER_COLOR: 'grey',
+  TABLE_SORT_ICON_COLOR: 'grey',
+  TABLE_DATA_BACKGROUND_IMAGE: 'grey',
+  TABLE_DATA_BACKGROUND_SIZE: 'grey',
+  ARROW_COLOR: '#D9D9D9',
+  LOG_AMOUNT_BACKGROUND: '#5276B7',
+  LOG_AMOUNT_COLOR: '#333',
+  LOG_WARN_AMOUNT_BACKGROUND: '#966C08',
+  LOG_ERROR_AMOUNT_BACKGROUND: '#DD3134',
+  LOG_DEBUG_AMOUNT_BACKGROUND: '#097BB3'
+};
+
+const style = {
+  BASE_FONT_FAMILY: 'Inconsolata, monospace'
+};
+
+const CONSOLE_FEED_LIGHT_ICONS = {
+  LOG_WARN_ICON: `url(${warnLightUrl})`,
+  LOG_ERROR_ICON: `url(${errorLightUrl})`,
+  LOG_DEBUG_ICON: `url(${debugLightUrl})`,
+  LOG_INFO_ICON: `url(${infoLightUrl})`,
+  LOG_COMMAND_ICON: `url(${commandLightUrl})`,
+  LOG_RESULT_ICON: `url(${resultLightUrl})`
+};
+
+const CONSOLE_FEED_DARK_ICONS = {
+  LOG_WARN_ICON: `url(${warnDarkUrl})`,
+  LOG_ERROR_ICON: `url(${errorDarkUrl})`,
+  LOG_DEBUG_ICON: `url(${debugDarkUrl})`,
+  LOG_INFO_ICON: `url(${infoDarkUrl})`,
+  LOG_COMMAND_ICON: `url(${commandDarkUrl})`,
+  LOG_RESULT_ICON: `url(${resultDarkUrl})`
+};
+
+const CONSOLE_FEED_CONTRAST_ICONS = {
+  LOG_WARN_ICON: `url(${warnContrastUrl})`,
+  LOG_ERROR_ICON: `url(${errorContrastUrl})`,
+  LOG_DEBUG_ICON: `url(${debugContrastUrl})`,
+  LOG_INFO_ICON: `url(${infoContrastUrl})`,
+  LOG_COMMAND_ICON: `url(${commandContrastUrl})`,
+  LOG_RESULT_ICON: `url(${resultContrastUrl})`
+};
+
+const getConsoleFeedStyle = (theme, fontSize) => {
+  const CONSOLE_FEED_SIZES = {
+    TREENODE_LINE_HEIGHT: 1.2,
+    BASE_FONT_SIZE: `${fontSize}px`,
+    ARROW_FONT_SIZE: `${fontSize}px`,
+    LOG_ICON_WIDTH: `${fontSize}px`,
+    LOG_ICON_HEIGHT: `${1.45 * fontSize}px`
+  };
+
+  switch (theme) {
+    case 'light':
+      return Object.assign(
+        CONSOLE_FEED_LIGHT_STYLES || {},
+        CONSOLE_FEED_LIGHT_ICONS,
+        CONSOLE_FEED_SIZES,
+        style
+      );
+    case 'dark':
+      return Object.assign(
+        CONSOLE_FEED_DARK_STYLES || {},
+        CONSOLE_FEED_DARK_ICONS,
+        CONSOLE_FEED_SIZES,
+        style
+      );
+    case 'contrast':
+      return Object.assign(
+        CONSOLE_FEED_CONTRAST_STYLES || {},
+        CONSOLE_FEED_CONTRAST_ICONS,
+        CONSOLE_FEED_SIZES,
+        style
+      );
+    default:
+      return '';
```

Comment:
This is supposed to return an object, right?  So `return '';` doesn't make sense.

Reply:
Yes!! I'm thinking of setting the default to the light theme instead?